### ISSUE TITLE
Global Chromium interception

### ISSUE THREAD

**Issue by tyilo** [httptoolkit/httptoolkit#621]
Currently fresh Chrome and global Chrome interceptions are available. However for Chromium only fresh Chromium is available, but not global Chromium.

It seems like the implementation for the global Chrome interception could easily be modified to also support global Chromium.

---

_Does this affect you too? Click below and add a :+1: to vote for this and help decide where HTTP Toolkit goes next, or [go vote on the other most popular ideas so far](https://github.com/httptoolkit/feedback/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)._

**Comment by pimterry** [comment]
Good idea! And conveniently quite easy to do, so I've actually just implemented and shipped it :smiley:

Your HTTP Toolkit will update in the background next time you open it, and the following time you should see a 'Global Chromium' option appear. Thanks for the suggestion!

**Comment by tyilo** [comment]
Thanks!
